### ISSUE TITLE
Config UI - Fix Type Mapping REQUEST Payload

### ISSUE THREAD

**Issue by e2corporation** [merico-dev/lake#672]
## Describe the bug
BE needs an updated request payload for `TypeMappings` (Currently using text-delimited string)

```
curl -v -XPOST "$LAKE_ENDPOINT/plugins/jira/sources" --data '
    {
        "name": "test-jira-source",
        "endpoint": "'"$JIRA_ENDPOINT"'",
        "basicAuthEncoded": "'"$JIRA_BASIC_AUTH_ENCODED"'",
        "epicKeyField": "'"$JIRA_ENDPOINT"'",
        "storyPointField": "'"$JIRA_ISSUE_STORYPOINT_FIELD"'",
        "storyPointCoefficient": '$JIRA_ISSUE_STORYPOINT_COEFFICIENT',
        "typeMappings": {
            "Story": {
                "standardType": "Requirement",
                "statusMappings": {
                    "已完成": {
                        "standardStatus": "Resolved"
                    },
                    "已解决": {
                        "standardStatus": "Resolved"
                    }
                }
            },
            "Incident": {
                "standardType": "Incident",
                "statusMappings": {
                    "已完成": {
                        "standardStatus": "Resolved"
                    }
                }
            },
            "Bug": {
                "standardType": "Bug",
                "statusMappings": {
                    "已完成": {
                        "standardStatus": "Resolved"
                    }
                }
            }
        }
    }' | jq
```

**Comment by joncodo** [comment]
Same as https://github.com/merico-dev/lake/issues/673

**Comment by e2corporation** [comment]
This should be closed, PR #676 was merged.
